docs/providers.rst

```diff
@@ -311,6 +311,9 @@ EthereumTesterProvider
     :class:`~eth_tester.backends.base.BaseChainBackend` class provided by the ``eth-tester`` library.
     If you would like a custom eth-tester instance to test with, see the
     ``eth-tester`` library `documentation <https://github.com/ethereum/eth-tester>`_ for details.
+    Note: to install needed the dependencies to use EthereumTesterProvider, you can install the
```

Comment:
If you want, you can highlight this as an "official" `Note` by using `.. NOTE:: `. Or we can leave it as text. 

Regardless, I think it should read: 
"to install _the needed_ dependencies..." instead of "to install needed the dependencies"

Reply:
fixed with a git commit --amend